docs/homepage.md

```diff
@@ -23,6 +23,12 @@ We do not store the secrets generated by this command and just rely on the secre
 
 ### Questions
 
+#### Why is this being added directly to the Signon landing page and via Publisher?
```

Comment:
Should this be "not via Publisher"?

Reply:
I meant to add and 'permissions via Publisher' but I think it has already been adressed so I will just keep it as Chris asked i.e. "Why is this being added directly to the Signon?"